### PR TITLE
reconstructor -> refabricator

### PR DESCRIPTION
**Pull request by Cry0flu1d**
is this a typo?

If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:
- [ ] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [ ] I have ensured that my code compiles, if applicable.
- [ ] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

File: core/assets/bundles/bundle.properties [hunk 1]
```diff
@@ -121,7 +121,7 @@ committingchanges = Committing Changes
 done = Done
 feature.unsupported = Your device does not support this feature.
 
-mods.initfailed = [red]⚠[] The previous Mindustry instance failed to initialize. This was likely caused by misbehaving mods.\n\nTo prevent a crash loop, [red]all mods have been disabled.[]
+mods.initfailed = [red]âš [] The previous Mindustry instance failed to initialize. This was likely caused by misbehaving mods.\n\nTo prevent a crash loop, [red]all mods have been disabled.[]
```
Comment (Weathercold) [inline core/assets/bundles/bundle.properties:124]: what was changed here

Comment (MEEPofFaith) [reply core/assets/bundles/bundle.properties:124]: Not sure, my bundle pr also had changes to random lines with no apparent difference

Comment (Cry0flu1d) [reply core/assets/bundles/bundle.properties:124]: <strike>I dont know</strike>

Comment (Cry0flu1d) [reply core/assets/bundles/bundle.properties:124]: > what was changed here

<strike>I dont really know</strike>

**Pull request merged**